docs/sources/menu.yaml

```diff
@@ -167,6 +167,19 @@
       name: Transformations
     - link: /panels/field-options/
       name: Field options
+      children:
+        - link: /panels/field-options/
+          name: Field options and overrides
+        - link: /panels/field-options/configure-all-fields/
+          name: Configure all fields
+        - link: /panels/field-options/configure-specific-fields/
+          name: Configure specific fields
+        - link: /panels/field-options/standard-field-options/
+          name: Standard field options
+        - link: /panels/field-options/table-field-options/
+          name: Table field options
+        - link: /panels/field-options/filter-table-columns/
+          name: Filter table columns
```

Comment:
The reason it has its own page now is that according to @hugohaggmark, the current functionality is going to be expanded in the near future.

Reply:
However, even as it stands, this standalone task is plenty long enough to justify a separate topic.

Reply:
Don't agree, column filters are part of task of configuring the table visualization. Not a seperate task or workflow. The table visualization page is the reference page for all table options, so do not understand why this option is not part of that page.

Reply:
Because making huge topics that hold everything related to one subject is not good documentation. Long pages are tiring to read through, difficult to navigate, and they are not friendly to web searches.

The best practice is to have task-based documentation, and when possible, to chunk information into manageable concept, task, and reference pieces.